docs/v1/P4Runtime-Spec.adoc

```diff
@@ -601,14 +601,19 @@ their write requests. gRPC provides authentication methods cite:[gRPCAuth] that
 should be deployed to prevent untrusted clients from creating channels, and thus
 from making changes or even reading the state of the server.
 
-=== Default Role
+=== Default Role and Default Device ID
 
 A controller can omit the role message in `MasterArbitrationUpdate`. This
 implies the "default role", which corresponds to "full pipeline access".
 This also implies that a default role has a `role_id` of `""` (default).
 If using a default role, all RPCs from the controller (e.g. `Write`) must
 leave the `role` unset.
 
+The "default device_id" corresponds to `0`. As mentioned [previously](#sec-default-valued-fields),
```

Comment:
nit: This is technically mentioned in a later section. Maybe just "As per section X, ..."?

Reply:
But skip this and instead do the thing below imo.